docs/en/source/understanding/related-projects.rst

```diff
@@ -1,4 +1,53 @@
 Related Projects
 ###########################
 
-Soon
\ No newline at end of file
+With Querido Diário handling the heavy and innovative work of collecting, storing, and making thousands of municipal gazettes available, get to know projects that already use our data.
+
+`Querido Diário\: Technologies in Education`_
+************************************************
+
+`Querido Diário\: Technologies in Education`_ is capable of indentifying official acts related to the world of technologies acquired by the basic education network at the municipality level in specific themes such as accessibility, connectivity, management, infrastructure,  artificial inteligence, robotics, etc.
```

Comment:
Só corrigindo alguns typos e também retirei o "the world of" pq acho que é uma figura de linguagem ("universo das tecnologias") que funciona em português, mas não tanto em Inglês.

```suggestion
`Querido Diário\: Technologies in Education`_ is capable of identifying official acts related to  technologies acquired by the basic education network at the municipality level in specific themes such as accessibility, connectivity, management, infrastructure, artificial intelligence, robotics, etc.
```